Simplenote/src/main/java/com/automattic/simplenote/authentication/SignInFragment.kt

```diff
@@ -0,0 +1,65 @@
+package com.automattic.simplenote.authentication
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import android.widget.TextView
+import androidx.lifecycle.ViewModelProvider
+import com.automattic.simplenote.R
+import com.automattic.simplenote.utils.NetworkUtils
+import com.automattic.simplenote.viewmodels.MagicLinkRequestUiState
+import com.automattic.simplenote.viewmodels.RequestMagicLinkViewModel
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class SignInFragment: MagicLinkableFragment() {
+
+    var viewModel: RequestMagicLinkViewModel? = null
+
+    private var loginWithPassword: TextView? = null
+
+    override fun inflateLayout(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view = inflater.inflate(R.layout.fragment_login, container, false)
+        loginWithPassword = view.findViewById(R.id.login_with_password_button)
+        loginWithPassword?.setOnClickListener { _ ->
+            activity?.let { act ->
+                val intent = Intent(act, SimplenoteCredentialsActivity::class.java)
+                intent.putExtra("EXTRA_IS_LOGIN", true)
+                this.startActivity(intent)
+                act.finish()
+            }
+        }
+        return view
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        viewModel = ViewModelProvider(this).get(RequestMagicLinkViewModel::class.java)
```

Comment:
This can be deleted if you use the property delegate `by viewModels()` (comment above).